Automattic/vip-go-mu-plugins
# Remove limit from `get_sites`, when running with the `--network` flag.

## Description

By default, `get_sites()` will only return, at most, 100 sites. On larger networks, with more than 100 sites, running `wp jetpack-start connect --network` would not get past these first hundred. By including the `number` argument, `get_sites()` will return all the sites in the network, making it  possible to connect all the existing sites. 

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

Run `wp jetpack-start connect --network` on a site with more than 100 subsites. See if the script tries to connect all the subsites.
